lib/sanitized.js

```diff
@@ -0,0 +1,14 @@
+function sanitized(array){
```

Comment:
Conventionally, the function name would be a present tense verb whose direct object is the argument.

"I want to sanitize the string `foo`" ==> `sanitize(foo)`